docs/resources/yum.md.erb

```diff
@@ -1,5 +1,6 @@
 ---
 title: About the yum Resource
+os: RHEL
```

Comment:
Will probebly be a few more here, like amazon linux and some others. Ill get you a list